draft-gpew-priv-ppm.md

```diff
@@ -712,25 +718,23 @@ the first aggregate request. Shall we exclude it in subsequent requests
 somehow?] The remainder of the structure is dedicated to VDAF-specific request
 parameters.
 
-In order to provide replay protection, the leader is required to send aggregate
-sub-requests in ascending order, where the ordering on sub-requests is
-determined by the algorithm defined in {{anti-replay}}. Specifically, the leader
-constructs its request such that:
-
-* each sub-request follows the previous sub-request; and
-* the first sub-request follows the last sub-request in the previous aggregate
-  request.
+In order to provide replay protection, the leader preprocesses the set of
+reports carried by the AggregateReq as described in {{anti-replay}}. Any reports
+filtered out by this procedure MUST be ignored.
```

Comment:
I find this confusing because the leader should be applying the anti-replay logic at the time of receiving uploads. By the time it gets to constructing an `AggregateReq`, it should already have discarded any reports with duplicate nonces by this point. It's also confusing to state that the leader should "preprocess the set of reports carried by the AggregateReq" because the leader constructs the `AggregateReq`.

Reply:
Addressed the latter request. For the former, an implementation of the leader might opt to wait to filter reports until it's ready to begin the aggregation flow. This makes sense if it wants to limit the amount of work it has to do for each upload request, where latency matters a lot more.